.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,16 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
```

Comment:
Dev container could be kept of left, I find them useful and it doesn't change anything

Reply:
Definitely don't want to add unnecessary files if we can avoid it 😅

Reply:
👍no merging this one then